rails_local_setup.rb

```diff
@@ -46,6 +58,13 @@
 puts 'Setting secret_key_base'
 source_file_string += "export SECRET_KEY_BASE=#{`openssl rand -hex 64`}\n"
 
+if !exclude_tcell
+  puts 'Configuring TCell'
+  `bin/configure_tcell.rb`
+else
+  puts "****\nYou are developing with TCell disabled--be sure to test that any third-party scripts or connection sources you add are also added to TCell\n****\n"
```

Comment:
I can see this biting us without revised practices.  E.g. engineer adds a third-party script, doesn't test with tCell enabled, then something breaks in staging or production.  Simple proposal to avoid that:

Engineers iterate without tCell, but at least manually test _with tCell enabled_ before opening PR.

Ideally this would be caught by UI tests, but given we don't run them all for every PR, I think this slightly more tedious manual process would lower risk.  I'm open to other ideas, just thought this lurker worth noting.